File: ResourceManager.cpp
#include <OpenHome/Private/Debug.h>
#include <OpenHome/Buffer.h>

#include "ResourceManager.h"

using namespace std;
using namespace OpenHome;
using namespace OpenHome::Media;
using namespace OpenHome::Net;

ResourceManager::ResourceManager(const Brx& aNamespace)
{
}

void ResourceManager::WriteResource(const Brx& aUriTail, TIpAddress aInterface, std::vector<char*>& aLanguageList, IResourceWriter& aResourceWriter)
{
    Log::Print(aUriTail);
}

File: OpenHome/Media/ResourceManager.cs
using System;
using System.IO;
using System.Collections.Generic;

using OpenHome.Net.Device;

namespace OpenHome.Media
{
    public class ResourceManager : IResourceManager
    {
        public ResourceManager(IResourceManager aResourceManager)
        {
            iResourceManager = aResourceManager;
        }

        public void WriteResource(string aUriTail, uint aIpAddress, List<string> aLanguageList, IResourceWriter aWriter)
        {
            iResourceManager.WriteResource(aUriTail, aIpAddress, aLanguageList, aWriter);
        }

        private IResourceManager iResourceManager;
    }
}

